tensorflow/compiler/mlir/hlo/lib/utils/hlo_utils.cc

```diff
@@ -163,5 +163,22 @@ int64_t getArgumentIndex(mlir::FuncOp op, Value value) {
   return arg.getArgNumber();
 }
 
+/// Computes the memory usage of the given allocations.
+std::pair<size_t, size_t> computeMemory(const std::vector<Value> &allocs){
+  // Padding in Byte
+  const size_t padding = 64;
```

Comment:
Please use a constant with name for magic numbers, so that they can be reused.

Reply:
Okey